CONTRIBUTING.md

```diff
@@ -121,6 +121,9 @@ then it must cover the following cases:
 - Success with the context, if any allowed in the operator signature
 - If an error is thrown
 
+#### <a id="unit-tests"></a>CI Tests
+- Set your [Travis](https://travis-ci.org/) account to point against your fork
```

Comment:
what about adding some more background explanation for these tests? below's few topics I can think of : 
- `travis` setup on forked repo will allow ci tests on forked repo without submitting PR to master
- `saucalabs` setup will allow run browser test on forked repo: due to saucelab doesn't allow browser test on PR, this will help verify test results before PR's checked in master
-  since master runs both of test per each check in, it'd be welcome to setup those test for creating PR